### PR TITLE
CHE-7278 Properly handle restarting the workspace

### PR DESCRIPTION
**Pull request by vitaliy-guliy**
### What does this PR do?
Handles restarting current workspace.

Workspace is stopping
stopping

Workspace stopped
stopped

Starting
starting

Running
running

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/7278

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterJsonRpcInitializer.java [hunk 1]
```diff
@@ -90,12 +92,16 @@ public WsMasterJsonRpcInitializer(
     eventBus.addHandler(
         WorkspaceStoppedEvent.TYPE,
         e -> {
-          unsubscribeFromEvents();
-          terminate();
+          //          unsubscribeFromEvents();
```
Comment (vparfonov) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterJsonRpcInitializer.java]: comment code

**Review by vparfonov — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterJsonRpcInitializer.java (hunk 1 above)
Comment (vitaliy-guliy) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterJsonRpcInitializer.java]: Removed

**Comment by vitaliy-guliy** [comment]
ci-build

**Comment by vitaliy-guliy** [comment]
ci-test

**Review by azatsarynnyy — APPROVED** [review]

**Review by musienko-maxim — APPROVED** [review]

**Review by ashumilova — APPROVED** [review]

**Pull request merged**